editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx

```diff
@@ -645,8 +695,10 @@ describe('Select Mode Double Clicking With Fragments', () => {
     await doubleClick()
     await doubleClick()
     await doubleClick()
+    await doubleClick()
 
     expect(renderResult.getEditorState().editor.selectedViews).toEqual([desiredPath])
+    setFeatureEnabled('Fragment support', false)
```

Comment:
these should store the original value for the 'Fragment support' feature switch, and restore that. other tests use `before`/`after` to achieve this.

see https://github.com/concrete-utopia/utopia/blob/6af6cdaea3803fbafcdaea16d726bcf2072328f6/editor/src/components/canvas/canvas-strategies/strategies/absolute-reparent-strategy.spec.browser2.tsx#L785-L792

Reply:
btw we use this pattern in a bunch of places, it could be extracted into a single helper function that takes a feature switch name and the desired temporary value for the duration of the test.